t/helpers/peers.go

```diff
@@ -0,0 +1,66 @@
+package helpers
+
+import (
+	"errors"
+	"time"
+
+	"github.com/ethereum/go-ethereum/p2p"
+	"github.com/ethereum/go-ethereum/p2p/discover"
+)
+
+var (
+	// ErrNoRunningNode node is not running.
+	ErrNoRunningNode = errors.New("there is no running node")
+	// ErrEmptyPeerURL provided peer URL is empty
+	ErrEmptyPeerURL = errors.New("empty peer url")
+)
+
+type peer interface {
```

Comment:
@adambabik as `IsRunning` is not needed anymore i've changed the input of waitForPeer, so it gets a *p2p.Server instead of the node itself, so we don't need anymore this interface. 

Basically we will be calling this method like:
```
errCh := helpers.WaitForPeerAsync(n.Server(), peerURL, p2p.PeerEventTypeAdd, time.Second*5)
```

Please check it on the new [commit](https://github.com/status-im/status-go/pull/1064/commits/32f1bb66889f8b7633b4ee383254eadb8f234b09)